src/collections.ts

```diff
@@ -7,6 +7,11 @@ type BaseCollection<CollectionType> = {
   id: string;
 };
 
+export type CollectionInput<CollectionType extends UR = UR> = CollectionType & {
```

Comment:
CollectionInput is also used for API responses, maybe better to call it `CollectionItem`?

Reply:
done